test/integration/install/test_install.py

```diff
@@ -214,6 +215,29 @@ def test_install_tt_specific_version(tt_cmd, tmp_path):
     assert re.search(r"Tarantool CLI version 2.1.2", start_output)
 
 
+@pytest.mark.slow
+def test_install_tt_incomplete_version(tt_cmd, tmp_path):
+    configPath = tmp_path / config_name
+    # Create test config.
+    with open(configPath, "w") as f:
+        f.write("env:\n  bin_dir:\n  inc_dir:\n")
+
+    # Install latest tt.
+    install_cmd = (tt_cmd, "--cfg", configPath, "install", "tt", "2.1")
```

Comment:
I think we can test many different scenarios here with different version formats and errors using `parametrize`.

Reply:
`tt install` is a takes much time, so it's ok to test with integration tests some basic scenarios, but remaining scenarios should be tested with lighter tests (unit-tests).

Reply:
I add another generic case for `tt install tt <version>` because this test works fast enough. 
And used `parametrize` for easy future expansion.